terraform/provider/provider.go

```diff
@@ -299,8 +299,10 @@ func (p *Provider) Configure(ctx context.Context, req tfsdk.ConfigureProviderReq
 		creds = append(creds, client.LoadIdentityFileFromString(string(decoded)))
 	}
 
-	log.WithField("dir", profileDir).WithField("name", profileName).Debug("Using profile as the default auth method")
-	creds = append(creds, client.LoadProfile(profileDir, profileName))
+	if profileDir != "" {
```

Comment:
We should still default to profile if no creds were provided. 
```suggestion
	if profileDir != "" || len(creds) == 0 {
```